### PR TITLE
bump github.com/iovisor/gobpf

### PR DESCRIPTION
**Pull request by grooverdan**
Seems the previous version no longer exists

error was:

go: github.com/lebauce/gobpf@v0.0.0-20190909090614-f9e9df81702a: unknown revision f9e9df81702a

I picked https://github.com/iovisor/gobpf/commit/a3c161dcff4fd8645c4dde22179d1e1f250e8e75 as the nearest commit and it seem useful enough.

I probably haven't done the module updates correctly.

### CODE REVIEW

File: ebpf/Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ all: clean docker-ebpf-build
 
 %.o: %.c
 	$(CLANG) \
-		-I$${GOPATH}/pkg/mod/github.com/lebauce/gobpf@v0.0.0-20190909090614-f9e9df81702a/elf \
+		-I$${GOPATH}/pkg/mod/github.com/lebauce/gobpf@v0.0.0-20191007163444-a3c161dcff4f/elf/include \
```
Comment (safchain) [inline ebpf/Makefile]: the last commit is 27ec169 in this repo

**Comment by lebauce** [comment]
You need to regenerate the `ebpf/statics/bindata.go` (this is the reason forthe skydive-compile-tests job to fail)

**Comment by lebauce** [comment]
run functional-tests-backend-orientdb

**Comment by lebauce** [comment]
run functional-tests-backend-orientdb

**Comment by grooverdan** [comment]
I tried regeneration and it failed and then I lost track of this. Thanks for updating it.

**Comment by lebauce** [comment]
run create-docker-images

**Comment by grooverdan** [comment]
Thanks @safchain

**Pull request merged**